test/service/test_repository_specimen.py

```diff
@@ -46,7 +46,6 @@ def assert_file_type_summaries(hit):
             self.assertTrue('donorOrganisms' in hit)
             self.assertTrue('organoids' in hit)
             self.assertTrue('cellSuspensions' in hit)
-            self.assertFalse('projectSummary' in hit)
```

Comment:
Is there any harm in leaving this line in? Maybe a comment could explain that it's been deprecated.